gemrb/core/KeyMap.cpp

```diff
@@ -120,9 +120,13 @@ bool KeyMap::InitializeKeyMap(const path_t& inifile, const ResRef& tablefile)
 //main gamecontrol
 bool KeyMap::ResolveKey(unsigned short key, int group) const
 {
-	// FIXME: key is 2 bytes, but we ignore one. Some non english keyboards won't like this.
 	char keystr[2] = {(char)key, 0};
-	Log(MESSAGE, "KeyMap", "Looking up key: {}({}) ", key, keystr);
+	if (key < 128) {
```

Comment:
this doesnt actually address the `FIXME` you removed. We ought to work with keycodes, not characters.

Reply:
No, it primarily fixes the crash when libfmt encounters some invalid UTF-8 byte.

I was really thinking if there is any case with keyboard codes > 255, but yes, I can restore the FIXME note.

Reply:
Restored